language server crashes on macOS

On macOS 10.14.6, Julia 1.1.1, plugin 0.12.0, I constantly got crash reports on a project of 1088 lines of Julia.
Error message:
```javascript
[[object Object]]Language client is not ready yet
$onExtensionRuntimeError @ /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:3267
Error: Language client is not ready yet
	at LanguageClient.sendNotification (/Users/qz/.vscode/extensions/julialang.language-julia-0.12.0/node_modules/vscode-languageclient/lib/client.js:1729)
	at /Users/qz/.vscode/extensions/julialang.language-julia-0.12.0/out/jlpkgenv.js:74
	at Generator.next (<anonymous>)
	at fulfilled (/Users/qz/.vscode/extensions/julialang.language-julia-0.12.0/out/jlpkgenv.js:4)
$onExtensionRuntimeError @ /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:3267
The Julia Language Server server crashed 5 times in the last 3 minutes. The server will not be restarted.
onDidNotificationChange @ /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:2393
```

Here you go:

macOS 10.14.6
julia version 1.1.1

> /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:3267 [[object Object]]Language client is not ready yet
> $onExtensionRuntimeError @ /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:3267
> /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:3267 Error: Language client is not ready yet
	> at LanguageClient.sendNotification (/Users/lgarron/.vscode/extensions/julialang.language-julia-0.12.0/node_modules/vscode-languageclient/lib/client.js:1729)
	> at /Users/lgarron/.vscode/extensions/julialang.language-julia-0.12.0/out/jlpkgenv.js:74
	> at Generator.next (<anonymous>)
	> at fulfilled (/Users/lgarron/.vscode/extensions/julialang.language-julia-0.12.0/out/jlpkgenv.js:4)
> $onExtensionRuntimeError @ /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:3267
> /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:240 [Extension Host] (node:4690) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
> t.log @ /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:240
> /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:3267 [[object Object]]Cannot read property 'indexOf' of undefined
> $onExtensionRuntimeError @ /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:3267
> /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:3267 TypeError: Cannot read property 'indexOf' of undefined
	> at getAbsolutePath (/Users/lgarron/.vscode/extensions/donjayamanne.jupyter-1.1.5/out/src/common/configSettings.js:57)
	> at PythonSettings.initializeSettings (/Users/lgarron/.vscode/extensions/donjayamanne.jupyter-1.1.5/out/src/common/configSettings.js:30)
	> at constructor.disposables.push.vscode.workspace.onDidChangeConfiguration (/Users/lgarron/.vscode/extensions/donjayamanne.jupyter-1.1.5/out/src/common/configSettings.js:18)
	> at u.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:48)
	> at p.$acceptConfigurationChanged (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:407)
	> at define.$acceptConfigurationChanged.getConfigProvider.then.n (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:406)
> $onExtensionRuntimeError @ /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:3267
> /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:2393 The Julia Language Server server crashed 5 times in the last 3 minutes. The server will not be restarted.
> onDidNotificationChange @ /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:2393

<img width="1072" alt="Screen Shot 2019-07-29 at 19 02 58" src="https://user-images.githubusercontent.com/248078/62094903-8f5fb080-b233-11e9-82de-d5460c833c40.png">

--- a/src/jlpkgenv.ts
+++ b/src/jlpkgenv.ts
@@ -31,7 +31,7 @@ export async function getProjectFilePaths(envpath: string) {
     }
 }
 
-async function switchEnvToPath(envpath: string) {
+async function switchEnvToPath(envpath: string, notifyLS: boolean) {
     g_path_of_current_environment = envpath;
 
     let section = vscode.workspace.getConfiguration('julia');
@@ -73,7 +73,9 @@ async function switchEnvToPath(envpath: string) {
         }
     }
 
-    g_languageClient.sendNotification("julia/activateenvironment", envpath);
+    if (notifyLS) {
+        g_languageClient.sendNotification("julia/activateenvironment", envpath);
+    }
 }
 
 async function changeJuliaEnvironment() {
@@ -131,7 +133,7 @@ async function changeJuliaEnvironment() {
                 let envPath = vscode.Uri.parse(envPathUri).fsPath;
                 let isThisAEnv = await fs.exists(path.join(envPath, 'Project.toml'));
                 if (isThisAEnv) {
-                    switchEnvToPath(envPath);
+                    switchEnvToPath(envPath, true);
                 }
                 else {
                     vscode.window.showErrorMessage('The selected path is not a julia environment.');
@@ -139,7 +141,7 @@ async function changeJuliaEnvironment() {
             }
         }
         else {
-            switchEnvToPath(resultPackage.description);
+            switchEnvToPath(resultPackage.description, true);
         }
     }
 }
@@ -197,7 +199,7 @@ export async function activate(context: vscode.ExtensionContext, settings: setti
     g_current_environment.text = "Julia env: [loading]";
     g_current_environment.command = "language-julia.changeCurrentEnvironment";
     context.subscriptions.push(g_current_environment);
-    await switchEnvToPath(await getEnvPath());
+    await switchEnvToPath(await getEnvPath(), false); // We don't need to notify the LS here because it will start with that env already
 }
 
 export function onDidChangeConfiguration(newSettings: settings.ISettings) {

